src/riak_kv_app.erl

```diff
@@ -155,6 +155,10 @@ start(_Type, _StartArgs) ->
                                           [true, false],
                                           false),
 
+            riak_core_capability:register({riak_kv, w1c_batch_vnode},
```

Comment:
If the batching of operations is transparent to the callers of riak_kv_ts_api:put_data (as I understand it is), then it is inconsequential whether a node has this capability or not. What are the circumstances where you expect this capability to be queried?